lib/utils/client/src/test/reporting-test.js

```diff
@@ -366,22 +364,33 @@ const tests = (secured) => {
 
     context('on business error', () => {
       context('on 409 response code', () => {
-        const testConflict = (errorResponseBody, failures) => {
+        const checkResponse = (error, response, expectedResponseBody, done) => {
+          expect(error).to.equal(undefined);
+          expect(response).to.deep.equal(expectedResponseBody);
+          done();
+        };
+
+        const testConflict = (errorResponseBody, failures, options) => {
           beforeEach((done) => {
             responseError = undefined;
             responseBody = generateResponse(409, {}, errorResponseBody);
 
             errors.consecutiveReportFailures = failures;
 
-            reporter.reportUsage(appUsage, token, (error, res) => {
-              expect(error).to.equal(undefined);
-              expect(res).to.equal(responseBody);
-              done();
-            });
+            if(isEmpty(options))
```

Comment:
This is done in order to do different checks for the two cases - 201 and 409 status codes.